### ISSUE TITLE
Thank you Mr. Kenny Kerr and Mr. Brent Rector for being so caring and responsive.

### ISSUE THREAD

**Issue by Noemata** [Microsoft/cppwinrt#228]
Of the many Microsoft repositories I snoop on, the cppwinrt is one of the best because of the articulate, succinct, and enthusiastic responsiveness of Mr. Kenny Kerr and Mr. Brent Rector in particular.

A big thank you for the quality of your work and communications over the years!

**Comment by VikingExplorer** [comment]
Yes, you guys are doing an awesome job.  Only 17 issues open is very good compared to say, CppRestSdk with 249 issues open, while depending on Boost 1.55 (1.64 is available), & hasn't even been updated for VS2017 or the latest compiler. Sad!

Been following Kenny Kerr for years, so glad when he joined MS.
